test/wallet-http-test.js

```diff
@@ -85,6 +92,45 @@ for (const witnessOpt of witnessOptions) {
       addr = Address.fromString(str, node.network);
     });
 
+    it('should change passphrase', async () => {
+      await wallet.setPassphrase('initial');
+      await wclient.lock('test');
+
+      // Incorrect Passphrase should not work
+      try {
+        await wallet.unlock('badpass');
+        throw new Error('Unlocked with INCORRECT passphrase!');
+      } catch(e) {
+        assert.strictEqual(e.message, 'Could not decrypt.', e.message);
+      }
+
+      // Correct Passphrase should work
+      try {
+        await wallet.unlock('initial');
+      } catch(e) {
+        throw new Error('Could not unlock with correct passphrase.');
+      }
+
+      await wallet.setPassphrase('newpass', 'initial');
+      await wclient.lock('test');
```

Comment:
I think this is locking a wallet called "test" ? I think it should be locking a wallet called `name`. However I noticed the test passes anyway, passes when this line is removed, and also passes when the correct wallet name is used... why is that? Is there something else we need to check here? Or does `setPassphrase` lock the wallet automatically? Maybe we should test _that_ ?

Reply:
Yes `setPassphrase` locks the wallet automatically, we do not require the additional locking statement